diff --git a/node_modules/@nuxtjs/tailwindcss/lib/module.js b/node_modules/@nuxtjs/tailwindcss/lib/module.js
index c935c0b..0352b28 100644
--- a/node_modules/@nuxtjs/tailwindcss/lib/module.js
+++ b/node_modules/@nuxtjs/tailwindcss/lib/module.js
@@ -56,17 +56,19 @@ module.exports = async function (moduleOptions) {
     */
     const { postcss } = nuxt.options.build
 
-    postcss.preset.stage = 1 // see https://tailwindcss.com/docs/using-with-preprocessors#future-css-features
-    postcss.plugins = postcss.plugins || {}
+    postcss.postcssOptions = postcss.postcssOptions || {}
+    postcss.postcssOptions.preset = postcss.postcssOptions.preset || {}
+    postcss.postcssOptions.preset.stage = 1 // see https://tailwindcss.com/docs/using-with-preprocessors#future-css-features
+    postcss.postcssOptions.plugins = postcss.postcssOptions.plugins || {}
 
     // Let modules extend the tailwind config
     await nuxt.callHook('tailwindcss:config', tailwindConfig)
 
     /* istanbul ignore if */
-    if (Array.isArray(postcss.plugins)) {
+    if (Array.isArray(postcss.postcssOptions.plugins)) {
       logger.error('Array syntax for postcss plugins is not supported with v3. Please use the object syntax: https://nuxtjs.org/guides/configuration-glossary/configuration-build#postcss')
-    } else if (typeof postcss.plugins === 'object') {
-      postcss.plugins.tailwindcss = tailwindConfig
+    } else if (typeof postcss.postcssOptions.plugins === 'object') {
+      postcss.postcssOptions.plugins.tailwindcss = tailwindConfig
     }
 
     /*
